src/Google.Api.Gax/Api/Gax/MetadataExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Grpc.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    internal static class MetadataExtensions
+    {
+        public static void Add(this Metadata metadata, IEnumerable<Metadata.Entry> items)
```

Comment:
Calling it Add allows it to be used as a collection initializer, as in the Clone method below